Sources/InstrumentationBaggage/Shims.swift

```diff
@@ -0,0 +1,20 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Service Context
+// open source project
+//
+// Copyright (c) 2020-2022 Apple Inc. and the Swift Service Context
+// project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@exported import ServiceContextModule
+
+public typealias Baggage = ServiceContext
+public typealias BaggageKey = ServiceContextKey
+public typealias AnyBaggageKey = AnyServiceContextKey
```

Comment:
`@available(*, deprecated)` can be used with `typealias` to provide warnings, so I think we can provide diagnostic messages here.

```suggestion
@available(*, deprecated, message: "Use 'ServiceContext' from 'ServiceContextModule' instead.")
public typealias Baggage = ServiceContext
@available(*, deprecated, message: "Use 'ServiceContextKey' from 'ServiceContextModule' instead.")
public typealias BaggageKey = ServiceContextKey
@available(*, deprecated, message: "Use 'AnyServiceContextKey' from 'ServiceContextModule' instead.")
public typealias AnyBaggageKey = AnyServiceContextKey
```

Emitting:
```
warning: 'Baggage' is deprecated: Use 'ServiceContext' from 'ServiceContextModule' instead.
```

Reply:
Yes those will become deprecated; I've but this up a bit early as a draft to check CI